lib/api/controllers/securityController.js

```diff
@@ -388,7 +390,19 @@ module.exports = function SecurityController (kuzzle) {
 
         return kuzzle.repositories.user.load(newRequestObject.data._id)
           .then(user => {
-            return kuzzle.repositories.user.persist(_.extend(user, newRequestObject.data.body), { database: { method: 'update' } });
```

Comment:
Yes, it's the way we are going to.

@j33f is implementing this.

So this PR will not be relevant after.